rabbitmq/json-queues.yml

```diff
@@ -0,0 +1,147 @@
+type: json
+data:
+  url: /api/queues
+  server: ${SPM_MONITOR_RABBITMQ_API_URL}
+  basicHttpAuthUsername: ${SPM_MONITOR_RABBITMQ_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${SPM_MONITOR_RABBITMQ_BASICAUTH_PASSWORD}
+
+observation:
+  - name: queues
+    metricNamespace: rabbitmq
+    path: $.[?(@.name=${name} && @.vhost=${vhost})]
+
+    metric:
+      - name: queues.consumers.active
+        source: active_consumers
+        type: long_gauge
+        label: active consumers
+        description: Number of active consumers, consumers that can immediately receive any messages sent to the queue
+
+      - name: queues.consumers
+        source: consumers
+        type: long_gauge
+        label: consumers
+        description: Number of consumers
+
+      - name: queues.consumer.utilisation
+        source: consumer_utilisation
+        type: long_gauge
```

Comment:
If it is ratio, perhaps it should be double, not long?

Reply:
Fixed, i change to gauge.